src/main/resources/crafter/studio/upgrade/3.1.x/3.1.0.34/system/sample-site-config-tools.xslt

```diff
@@ -51,22 +51,6 @@
         </xsl:copy>
     </xsl:template>
 
-    <xsl:template match="config/tools/tool/datasources">
```

Comment:
Why are we removing this?

Reply:
duplicate code. if you check the whole file you will see code block repeated